src/libraries/System.Globalization/tests/CultureInfo/CultureInfoEnglishName.cs

```diff
@@ -12,7 +12,9 @@ public static IEnumerable<object[]> EnglishName_TestData()
         {
             yield return new object[] { CultureInfo.CurrentCulture.Name, CultureInfo.CurrentCulture.EnglishName };
             yield return new object[] { "en-US", "English (United States)" };
-            yield return new object[] { "fr-FR", "French (France)" };
+            if (PlatformDetection.IsNotBrowser)
+                // Browser's ICU returns "French (FR)" instead for "fr-FR"
```

Comment:
Would it make more sense for this and the `CultureInfoNativeName.cs` file change below to still test this on the Browser? For example:

```C#
if (PlatformDetection.IsNotBrowser)
{
    yield return new object[] { "fr-FR", "French (France)" };
}
else
{
    // Browser's ICU returns "French (FR)" instead for "fr-FR"
    yield return new object[] { "fr-FR", "French (FR)" };
}
```

Reply:
Good idea, thanks!